File: php/scripts/get_safety_measures.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$safetyMeasure = $travelperk->travelSafe()->travelSafe()->airlineSafetyMeasures(
	"LH",
);
write_output("safety_measures", [
	$safetyMeasure->airline->name,
	$safetyMeasure->airline->iataCode,
	count($safetyMeasure->safetyMeasures),
	$safetyMeasure->safetyMeasures[0]->category->id,
	$safetyMeasure->safetyMeasures[0]->category->name,
	$safetyMeasure->safetyMeasures[0]->subCategory->id,
	$safetyMeasure->safetyMeasures[0]->subCategory->name,
	$safetyMeasure->safetyMeasures[0]->details,
	$safetyMeasure->safetyMeasures[0]->summary,
	$safetyMeasure->infoSource->name,
	$safetyMeasure->infoSource->url,
	$safetyMeasure->updatedAt,
]);

File: python/scripts/get_safety_measures.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
safety_measure = travelperk.travelsafe().travelsafe().airline_safety_measures(
    "LH"
)
write_output("safety_measures", [
    safety_measure.airline.name,
    safety_measure.airline.iata_code,
    len(safety_measure.safety_measures),
    safety_measure.safety_measures[0].category.id,
    safety_measure.safety_measures[0].category.name,
    safety_measure.safety_measures[0].sub_category.id,
    safety_measure.safety_measures[0].sub_category.name,
    safety_measure.safety_measures[0].details,
    safety_measure.safety_measures[0].summary,
    safety_measure.info_source.name,
    safety_measure.info_source.url,
    safety_measure.updated_at,
])
